source/guide_jellyfin.rst

```diff
@@ -0,0 +1,176 @@
+
+You can use Jellyfin alongside :lab:`Syncthing <guide_syncthing>`, to load your media onto Uberspace for streaming.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  - :manual:`supervisord <daemons-supervisord>`
+  - :manual:`domains <web-domains>`
+
+License
+=======
+
+The software is licensed under `GNU General Public License v2.0`_. All relevant information can be found in the GitHub_ repository of the project.
+
+Prerequisites
+=============
+
+Your dashboard URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Create a directory in ``/home/isabell`` for jellyfin and its files.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ mkdir ~/jellyfin
+  [isabell@stardust ~]$
+
+Download the combined archive from release_, including both the server and WebUI, as we will need both of them.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ wget -O ~/jellyfin/jellyfin.tar.gz https://repo.jellyfin.org/releases/server/linux/stable/combined/jellyfin_10.7.7_amd64.tar.gz
+  [isabell@stardust ~]$
+
+Extract the archive. Delete the archive post extraction.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ tar -xvzf ~/jellyfin/jellyfin.tar.gz  -C jellyfin
+  [isabell@stardust ~]$ rm ~/jellyfin/jellyfin.tar.gz
+  [isabell@stardust ~]$
+
+Create a symbolic link to the Jellyfin 10.7.7 directory. This allows an upgrade by repeating the above steps and enabling it by simply re-creating the symbolic link to the new version.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ ln -s ~/jellyfin/jellyfin_10.7.7 ~/jellyfin/jellyfin
+  [isabell@stardust ~]$
+
+Create four sub-directories for Jellyfin data, and one to store your media library.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$  mkdir data cache config log library
+  [isabell@stardust ~]$
+
+Create a small script to run Jellyfin, ``jellyfin.sh`` in ``~/jellyfin`` with the following content.
+
+.. code-block:: sh
+  :emphasize-lines: 2
+
+  #!/bin/bash
+  JELLYFINDIR="/home/isabell/jellyfin"
+
+  $JELLYFINDIR/jellyfin/jellyfin \
+  -d $JELLYFINDIR/data \
+  -C $JELLYFINDIR/cache \
+  -c $JELLYFINDIR/config \
+  -l $JELLYFINDIR/log
+
+Make the startup script above executable.
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$  chmod u+x ~/jellyfin/jellyfin.sh
+  [isabell@stardust ~]$
+
+Configuration
+=============
+
+Configure the web server
+------------------------
+
+.. note::
+    Jellyfin will run on port 8096.
+
+.. include:: includes/web-backend.rst
+
+Set up the daemon
+-----------------
+
+To start Jellyfin automatically and run it in the background, create ``~/etc/services.d/jellyfin.ini`` with the following content:
+
+.. code-block:: ini
+
+  [program:jellyfin]
+  command=bash %(ENV_HOME)s/jellyfin/jellyfin.sh
+
+.. include:: includes/supervisord.rst
+
+.. note::
+
+  If `jellyfin` is not ``RUNNING``, check your configuration and the logs using ``supervisorctl maintail``.
```

Comment:
Still not working for me:

```
[moritz@stardust jellyfin]$ supervisorctl maintail
a child quit
2021-11-25 12:06:09,213 INFO spawned: 'jellyfin' with pid 12978
2021-11-25 12:06:09,220 DEBG 'jellyfin' stderr output:
/home/moritz/jellyfin/jellyfin.sh: line 5: /home/isabell/jellyfin/jellyfin/jellyfin: No such file or directory

2021-11-25 12:06:09,220 DEBG fd 9 closed, stopped monitoring <POutputDispatcher at 139918775711288 for <Subprocess at 139918774909968 with name jellyfin in state STARTING> (stderr)>
2021-11-25 12:06:09,220 DEBG fd 7 closed, stopped monitoring <POutputDispatcher at 139918775710928 for <Subprocess at 139918774909968 with name jellyfin in state STARTING> (stdout)>
2021-11-25 12:06:09,220 INFO exited: jellyfin (exit status 127; not expected)
2021-11-25 12:06:09,220 DEBG received SIGCLD indicating a child quit
2021-11-25 12:06:12,443 INFO spawned: 'jellyfin' with pid 13027
2021-11-25 12:06:12,449 DEBG 'jellyfin' stderr output:
/home/moritz/jellyfin/jellyfin.sh: line 5: /home/isabell/jellyfin/jellyfin/jellyfin: No such file or directory

2021-11-25 12:06:12,450 DEBG fd 9 closed, stopped monitoring <POutputDispatcher at 139918775447776 for <Subprocess at 139918774909968 with name jellyfin in state STARTING> (stderr)>
2021-11-25 12:06:12,450 DEBG fd 7 closed, stopped monitoring <POutputDispatcher at 139918775682256 for <Subprocess at 139918774909968 with name jellyfin in state STARTING> (stdout)>
2021-11-25 12:06:12,450 INFO exited: jellyfin (exit status 127; not expected)
2021-11-25 12:06:12,450 DEBG received SIGCLD indicating a child quit
2021-11-25 12:06:12,450 INFO gave up: jellyfin entered FATAL state, too many start retries too quickly
```

Reply:
Ah, shell script needs to be changed to specify _your_ home directory. 
```/home/moritz/jellyfin/jellyfin.sh: line 5: /home/isabell/jellyfin/jellyfin/jellyfin: No such file or directory```
It is currently pointed to the default dummy user `isabell` in the shell script.

Reply:
I had just highlighted the line to be updated as per the style guide. Should I explicitly call out the change while in there?

Reply:
Just use `$USER`, no need to set your own username, that's unix built in ☺️

Reply:
Aha! Fair.